docs/introduction/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Documentation
+headline: Documentation
+bodyclass: docs
+layout: docs
+---
+# Development Process
+
+Building a solution based on Spine Event Engine framework is an iterative process which consists
+of the stages described in this document.
+ 
+## Getting domain knowledge
+
+The purpose of this step is to find out what we're going to build and why.
+Consider using [EventStorming](https://eventstorming.com) or other DDD-based analysis 
+methodology for helping you grasp the domain knowledge from the experts.
```

Comment:
As I understand, you have omitted `Ubiquitous Language` to ease the way in. However, we should mention it someday. Let's discuss later.

Reply:
Added issue #243 on this.